pom.xml

```diff
@@ -267,16 +255,42 @@
 			<scope>test</scope>
 		</dependency>
 
+		<!-- JAXB dependencies required from java9 onwards -->
```

Comment:
are you sure that we need these imports (the library does not provide any jaxb services).

Reply:
these dependencies are not used by the project, but restrain jersey to throw warnings about classes that were deprecated with java 9. jersey is a dependency included by junit-docker-rule.

as a solution, I set those dependencies to scope test, which also fixes the warnings.